gravitee-apim-plugin/gravitee-apim-plugin-endpoint/gravitee-apim-plugin-endpoint-http-proxy/src/main/java/io/gravitee/plugin/endpoint/http/proxy/HttpProxyEndpointConnector.java

```diff
@@ -68,21 +71,21 @@ public Set<ConnectorMode> supportedModes() {
 
     @Override
     public Completable connect(ExecutionContext ctx) {
-        return Completable.defer(() -> {
```

Comment:
Removing the defer make the following code executed when building the chain instead of during the subscription. Request being mutable I don't think this is a good idea.